data.sql

```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+-- INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, "Agumon", "2020-02-03", 0, true, 48);
+INSERT INTO animals VALUES (1, "Agumon", "2020-02-03", 0, true, 48);
+INSERT INTO animals VALUES (2, "Gabumon", "2020-11-15", 2, true, 8);
+INSERT INTO animals VALUES (3, "Pikachu", "2021-01-07", 1, false, 15.0);
+INSERT INTO animals VALUES (4, "Devimon", "2017-05-12", 5, true, 11);
+
```

Comment:
Great job @AbigiyaTY 🤝

- [ ] Kindly note that the value for the `weight_kg` is `10.23` not `48`. You may please resolve this.
- [ ] You may also maintain the `decimal` type or `float` for all `weight_kg` values instead of being `integers`.